es/introduction.md

```diff
@@ -20,25 +20,25 @@ El software es una infraestructura pública.
 
 En el siglo XXI el software puede ser considerado una infraestructura pública vital. Cada vez más, no es sólo la expresión de la política existente, sino el creador de nuevas políticas, por ejemplo, cuando los algoritmos deciden qué distritos necesitan servicios sociales o policiales adicionales.
 
-La mecánica de los programas informáticos, los algoritmos y la recogida de datos se ha convertido en un elemento clave en la ejecución de las políticas públicas. El código informático ejecuta ahora políticas que han sido programadas en el código legal a través de procedimientos democráticos. Ambas formas de código establecen las condiciones para que la sociedad funcione de acuerdo con los valores públicos fijados democráticamente, el último ejecutado por humanos, el primero por máquinas. En otras palabras, el código del software ha empezado a equivaler cada vez más al código legal.
+La mecánica de los programas informáticos, los algoritmos y la recolección de datos se han convertido en un elemento clave para la ejecución de las políticas públicas. El código informático ejecuta ahora políticas que han sido programadas en el código legal a través de procedimientos democráticos. Ambas formas de código establecen las condiciones para que la sociedad funcione de acuerdo con los valores públicos fijados democráticamente; el último ejecutado por humanos, el primero por máquinas. En otras palabras, el código del software ha empezado a equivaler cada vez más al código legal.
```

Comment:
Don't mind in between "recogida" o "recolección". Both of them are singular so it should be "ha convertido" ("(...) la recolección (...) se ha convertido..."). Don't mind either in between "en" or "para".

The last ";" could be substituted by a normal comma.